videointelligence/video_analyze/gen/template.go

```diff
@@ -179,25 +179,19 @@ func speechTranscription__SUFFIX__(w io.Writer, file string) error {
 		// Each alternative is a different possible transcription
 		// and has its own confidence score.
 		for _, alternative := range transcription.GetAlternatives() {
-			transcript := alternative.GetTranscript()
-			confidence := alternative.GetConfidence()
-			words := alternative.GetWords()
-
 			fmt.Fprintf(w, "Alternative level information:\n")
-			fmt.Fprintf(w, "\tTranscript: %v\n", transcript)
-			fmt.Fprintf(w, "\tConfidence: %v\n", confidence)
+			fmt.Fprintf(w, "\tTranscript: %v\n", alternative.GetTranscript())
+			fmt.Fprintf(w, "\tConfidence: %v\n", alternative.GetConfidence())
 
 			fmt.Fprintf(w, "Word level information:\n")
-			for _, wordInfo := range words {
-				word := wordInfo.GetWord()
+			for _, wordInfo := range alternative.GetWords() {
 				startTime := wordInfo.GetStartTime()
 				endTime := wordInfo.GetEndTime()
-				speakerTag := wordInfo.GetSpeakerTag()
 				fmt.Fprintf(w, "\t%4.1f - %4.1f: %v (speaker %v)\n",
-					float64(startTime.GetSeconds())+float64(startTime.GetNanos())*1e-9,
-					float64(endTime.GetSeconds())+float64(endTime.GetNanos())*1e-9,
-					word,
-					speakerTag)
+					float64(startTime.GetSeconds())+float64(startTime.GetNanos())*1e-9, // start as seconds
```

Comment:
Would you mind moving this formatting to the `startTime` and `endTime` declarations? Then this `Fprintf` call can all be on one line.

Reply:
It's still a fairly long line, I would still break it into multiple lines.

Original:
```
for _, wordInfo := range alternative.GetWords() {
	startTime := wordInfo.GetStartTime()
	endTime := wordInfo.GetEndTime()
	fmt.Fprintf(w, "\t%4.1f - %4.1f: %v (speaker %v)\n",
		float64(startTime.GetSeconds())+float64(startTime.GetNanos())*1e-9, // start as seconds
		float64(endTime.GetSeconds())+float64(endTime.GetNanos())*1e-9,     // end as seconds
		wordInfo.GetWord(),
		wordInfo.GetSpeakerTag())
}
```

Option 1:
```
for _, wordInfo := range alternative.GetWords() {
	startSec := float64(wordInfo.GetStartTime().GetSeconds()) + float64(wordInfo.GetStartTime().GetNanos())*1e-9
	endSec := float64(wordInfo.GetEndTime().GetSeconds()) + float64(wordInfo.GetEndTime().GetNanos())*1e-9
	fmt.Fprintf(w, "\t%4.1f - %4.1f: %v (speaker %v)\n", startSec, endSec, wordInfo.GetWord(), wordInfo.GetSpeakerTag())
}
```

Option 2:
```
for _, wordInfo := range alternative.GetWords() {
	startTime := wordInfo.GetStartTime()
	endTime := wordInfo.GetEndTime()
	startSec := float64(startTime.GetSeconds()) + float64(startTime.GetNanos())*1e-9
	endSec := float64(endTime.GetSeconds()) + float64(endTime.GetNanos())*1e-9
	fmt.Fprintf(w, "\t%4.1f - %4.1f: %v (speaker %v)\n", startSec, endSec, wordInfo.GetWord(), wordInfo.GetSpeakerTag())
}
```

I think the original is still the most readable in this case, it's pretty common to break long function calls into lines like that in Python too. I also explored creating a lambda function to do the formatting but that required another import on the `Duration` proto, so I decided to not do it that way in favor of sample simplicity.